### PR TITLE
SwitcherButton for network/wallet switcher

### PR DESCRIPTION
**Pull request by ShawTim**

### CODE REVIEW

File: packages/tempus-client_v3/src/components/shared/SwitcherButton/SwitcherButton.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  label?: string;
+  title?: string;
+  selected: boolean;
+  onClick: () => void;
+}
+
+const SwitcherButton: FC<SwitcherButtonProps> = props => {
+  const { logoType, label, title, selected, onClick } = props;
+
+  const weight = useMemo(() => (selected ? 'bold' : 'regular'), [selected]);
```
Comment (lazarmitic) [inline packages/tempus-client_v3/src/components/shared/SwitcherButton/SwitcherButton.tsx:21]: I think we should not use `useMemo` for simple boolean checks. In this case `useMemo` is probably slowing app more then it is speeding it up.

File: packages/tempus-client_v3/src/components/shared/SwitcherButton/switcher-button.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .tc__switcher-btn__label {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+
+    svg {
```
Comment (danieledamiani) [inline packages/tempus-client_v3/src/components/shared/SwitcherButton/switcher-button.scss:17]: from the `spec` file it seems `svg` is not a descendant of `.tc__switcher-btn__label`

File: packages/tempus-client_v3/src/components/shared/SwitcherButton/SwitcherButton.tsx (hunk 1 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/SwitcherButton/SwitcherButton.tsx:21]: good comment

File: packages/tempus-client_v3/src/components/shared/SwitcherButton/switcher-button.scss (hunk 2 above)
Comment (ShawTim) [reply packages/tempus-client_v3/src/components/shared/SwitcherButton/switcher-button.scss:17]: it is. there are 2 svg here. this one is for the svg with `.tc__switcher-btn__label` in specific, which is the checkmark icon

**Review by lazarmitic — APPROVED** [review]

**Review by danieledamiani — APPROVED** [review]

**Pull request merged**